packages/spor-datepicker-react/src/DateTimeSegment.tsx

```diff
@@ -33,7 +33,7 @@ export const DateTimeSegment = ({ segment, state }: DateTimeSegmentProps) => {
       borderRadius="xs"
       color={
         segment.isPlaceholder
-          ? "osloGrey"
+          ? "dimGrey"
```

Comment:
Det var litt for dårlig placeholder-kontrast. Viser seg at placeholder-tekst også er underlagt samme krav til kontrast som vanlig tekst.